README.md

```diff
@@ -2,6 +2,10 @@
 
 Tracks github git tags (Annotated or Lightweight) regardless of the branch.
 
+## IMPORTANT
+
+**Be aware that this resource only supports tags in line with [Semver](https://semver.org/)**
```

Comment:
It should be mentioned that invalid tags will be discarded and will not crash the resource.